lib/northbound.c

```diff
@@ -679,6 +695,11 @@ int nb_candidate_edit(struct nb_config *candidate,
 						   ly_native_ctx, dep_xpath,
 						   NULL, LYD_NEW_PATH_UPDATE,
 						   &dep_dnode);
+				/* Create default nodes */
+				if (!err)
+					err = lyd_new_implicit_tree(
```

Comment:
Why are we creating a new tree here?

Reply:
The "_tree" suffix is used in ly2 to is indicate the action is happening over the tree, in this case that the creation of the implicit (default) nodes happens over the tree, not that it's creating a new tree.

Reply:
ok got it.